### PR TITLE
Rui/update documentation

### PR DESCRIPTION
**Pull request by rhanIDM**
@celiot-IDM, please review ChangeRateLimits section and confirm that implementation logic was clearly explained.
@BHagedorn-IDM & @JSchripsema-IDM, please feel free to directly edit the paragraphs that changed.

### CODE REVIEW

File: pacehrh/vignettes/pacehrh.rmd [hunk 1]
```diff
@@ -185,6 +184,16 @@ one used as an input to the model, so values can be changed directly here if you
...
+* **Max**: The upper limit of permissible cumulative change in rates, expressed as a ratio to the rate values at the beginning of the simulation. **Max** must be greater than 0. For example, with a **Max** value of 1.1 set for the rate category "Mortality", the initial mortality rates are multiplied by 1.1 to establish upper limits for predicted mortality rates over the course of the simulation. Any predicted mortality rates above these upper limits would be replaced by the upper limits by the model. If **Max** is left empty for any **RateCategory**, the model would run without imposing an upper limit on cumulative change for this **RateCategory**.For each **RateCategory**, the value set for **Max** must be greater than or equal to the value set for **Min**.
```
Comment (celiot-IDM) [inline pacehrh/vignettes/pacehrh.rmd:195]: Missing space before the final sentence in the para.

File: pacehrh/vignettes/pacehrh.rmd [hunk 2]
```diff
@@ -233,6 +242,18 @@ to the model. Columns G through I do not affect the running of the model, but ar
...
 * **HoursPerWeek**: The number of hours per week a healthcare workers spends on a task, usually a non-clinical task that is not directly related to population rates. For example, there may be a few hours of weekly administrative tasks required, regardless of clinical care provided. Community engagement, outreach, and supply management may also be tasks with this dynamic.
 
+### CoverageRates
+
+The **CoverageRates** sheets specify the service coverage rates for healthcare tasks from the beginning year of the simulation to the ending year of the simulation. The user has the flexibility to specify coverage rates for as many or few tasks as they prefer. Each row represents a healthcare task. Columns A and B reference the healthcare tasks as listed in **TaskValues** sheets. In columns C through W, the user enters any value between zero and one, which is used as a fixed coverage level for that task in the year indicated by the column name. The default sheet max out at 20 years, since extrapolation beyond that is a questionable choice. If a task is missing from the sheet, or the values are not filled in, the model will assume 100% coverage for that year-task combination.
```
Comment (celiot-IDM) [inline pacehrh/vignettes/pacehrh.rmd:247]: The R code doesn't do anything to enforce this 20-year limit. Is that a problem? (The comment about the danger of simulating beyond 20 years might be enough.)

**Review by celiot-IDM — APPROVED** [review]
A couple of minor issues, but nothing that should stop merge.

**Review by JSchripsema-IDM — APPROVED** [review]
Looks great! Charles already caught the missing space I noticed.

**Comment by celiot-IDM** [comment]
I've fixed the missing space. I'm going to leave the 20-year limit text alone. Will merge as soon as the build tests complete.

File: pacehrh/vignettes/pacehrh.rmd (hunk 2 above)
Comment (celiot-IDM) [reply pacehrh/vignettes/pacehrh.rmd:247]: Not enough of a problem to stop the merge.

File: pacehrh/vignettes/pacehrh.rmd (hunk 1 above)
Comment (celiot-IDM) [reply pacehrh/vignettes/pacehrh.rmd]: Fixed.

**Pull request merged**